test/parser/grouped.ics

```diff
@@ -0,0 +1,3 @@
+BEGIN:VCALENDAR
+GROUP1.X-TEST;VALUE=TEXT:test!
```

Comment:
This isn't valid iCal but it does prove the handling difference between iCal and vCard.